x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/node_allocation/node_allocation.js

```diff
@@ -25,15 +28,16 @@ const learnMoreLinks = (
     />
   </Fragment>
 );
-class NodeAllocationUi extends Component {
+
+export class NodeAllocation extends Component {
```

Comment:
TODO: Confirm this looks correct in the UI.

Reply:
![image](https://user-images.githubusercontent.com/1238659/59351348-116f3680-8cd3-11e9-986b-6c98708eae3a.png)